### PR TITLE
DOCSUP-8233: Edit and translate to Russian

### PR DESCRIPTION
**Pull request by sevirov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation (edit and translate to Russian)

### CODE REVIEW

File: docs/ru/operations/system-tables/replication_queue.md [hunk 1]
```diff
@@ -14,7 +14,17 @@
 
 -   `node_name` ([String](../../sql-reference/data-types/string.md)) — имя узла в ZooKeeper.
 
--   `type` ([String](../../sql-reference/data-types/string.md)) — тип задачи в очереди: `GET_PARTS`, `MERGE_PARTS`, `DETACH_PARTS`, `DROP_PARTS` или `MUTATE_PARTS`.
+-   `type` ([String](../../sql-reference/data-types/string.md)) — тип задачи в очереди:
+
+    -   `GET_PART` — взять кусок из другой реплики.
+    -   `ATTACH_PART` — присоединить кусок. Задача может быть выполнена и с куском из нашей собственной реплики (если она находится в папке `detached`). Эта задача практически идентична задаче `GET_PART`, лишь немного оптимизирована.
```
Comment (olgarev) [inline docs/ru/operations/system-tables/replication_queue.md:20]:
```suggestion
    -   `ATTACH_PART` — присоединить кусок. Задача может быть выполнена и с куском из нашей собственной реплики (если он находится в папке `detached`). Эта задача практически идентична задаче `GET_PART`, лишь немного оптимизирована.
```

File: docs/ru/sql-reference/statements/alter/partition.md [hunk 2]
```diff
@@ -83,17 +83,22 @@ ALTER TABLE visits ATTACH PART 201901_2_2_0;
...
 
-Этот запрос реплицируется. Реплика-иницатор проверяет, есть ли данные в директории `detached`. Если данные есть, то запрос проверяет их целостность. В случае успеха данные добавляются в таблицу. Все остальные реплики загружают данные с реплики-инициатора запроса.
+Этот запрос реплицируется. Реплика-иницатор проверяет, есть ли данные в директории `detached`.
+Если данные есть, то запрос проверяет их целостность. В случае успеха данные добавляются в таблицу.
 
-Это означает, что вы можете разместить данные в директории `detached` на одной реплике и с помощью запроса `ALTER ... ATTACH` добавить их в таблицу на всех репликах.
+Если не инициаторная реплика, получив команду присоединения, находит кусок с правильными контрольными суммами в своей собственной папке `detached`, она присоединяет данные, не извлекая их из других реплик.
+Если нет куска с правильными контрольными суммами, данные загружаются из любой реплики, имеющей этот кусок.
+
+Вы можете разместить данные в директории `detached` на одной реплике и с помощью запроса `ALTER ... ATTACH` добавить их в таблицу на всех репликах.
```
Comment (olgarev) [inline docs/ru/sql-reference/statements/alter/partition.md:92]:
```suggestion
Вы можете поместить данные в директорию `detached` на одной реплике и с помощью запроса `ALTER ... ATTACH` добавить их в таблицу на всех репликах.
```

File: docs/ru/sql-reference/statements/system.md [hunk 3]
```diff
@@ -220,41 +222,49 @@ SYSTEM START FETCHES [[db.]replicated_merge_tree_family_table_name]
...
 
 ### SYNC REPLICA {#query_language-system-sync-replica}
+
 Ждет когда таблица семейства `ReplicatedMergeTree` будет синхронизирована с другими репликами в кластере, будет работать до достижения `receive_timeout`, если синхронизация для таблицы отключена в настоящий момент времени:  
 
 ``` sql
 SYSTEM SYNC REPLICA [db.]replicated_merge_tree_family_table_name
 ```
 
+Таблица `[db.]replicated_merge_tree_family_table_name` синхронизирует команды из общего реплицированного лога в свою собственную очередь репликации. Затем запрос ждет, пока реплика не обработает все синхронизированные команды.
```
Comment (olgarev) [inline docs/ru/sql-reference/statements/system.md:264]:
```suggestion
После выполнения этого запроса таблица `[db.]replicated_merge_tree_family_table_name` синхронизирует команды из общего реплицированного лога в свою собственную очередь репликации. Затем запрос ждет, пока реплика не обработает все синхронизированные команды.
```

**Comment by sevirov** [comment]
@tavplubix Please, review this PR.

File: docs/ru/sql-reference/statements/system.md [hunk 4]
```diff
@@ -263,5 +273,5 @@ SYSTEM RESTART REPLICA [db.]replicated_merge_tree_family_table_name
 ```
 
 ### RESTART REPLICAS {#query_language-system-restart-replicas}
-Реинициализация состояния Zookeeper сессий для всех `ReplicatedMergeTree` таблиц, сравнивает текущее состояние с тем что хранится в Zookeeper как источник правды и добавляет задачи Zookeeper очередь если необходимо
 
+Реинициализация состояния Zookeeper сессий для всех `ReplicatedMergeTree` таблиц, сравнивает текущее состояние с тем что хранится в Zookeeper как источник правды и добавляет задачи Zookeeper очередь если необходимо
```
Comment (tavplubix) [inline docs/ru/sql-reference/statements/system.md:277]:
```suggestion
Реинициализация состояния ZooKeeper-сессий для всех `ReplicatedMergeTree` таблиц. Сравнивает текущее состояние реплики с тем, что хранится в ZooKeeper, как c источником правды, и добавляет задачи в очередь репликации в  ZooKeeper если необходимо
```

**Comment by sevirov** [comment]
@tavplubix I fixed. Please, review again.

**Review by tavplubix — APPROVED** [review]

**Pull request merged**